--- a/src/main/java/net/pms/dlna/MediaInfo.java
+++ b/src/main/java/net/pms/dlna/MediaInfo.java
@@ -46,15 +46,8 @@
 			libraryName = "mediainfo";
 		}
 
-		// libmediainfo for Linux depends on libzen
 		if (!Platform.isWindows() && !Platform.isMac()) {
-			try {
-				// We need to load dependencies first, because we know where our native libs are (e.g. Java Web Start Cache).
-				// If we do not, the system will look for dependencies, but only in the library path.
-				NativeLibrary.getInstance("zen");
-			} catch (LinkageError e) {
-				LOGGER.warn("Error loading libzen: " + e.getMessage());
-			}
+			NativeLibrary.addSearchPath(libraryName, "/usr/lib/ps3mediaserver/");
 		}
 	}
 
