Description: Remove unstable warning in window title
--- a/src/main/java/net/pms/newgui/LooksFrame.java
+++ b/src/main/java/net/pms/newgui/LooksFrame.java
@@ -266,11 +266,6 @@
 		String projectVersion = PropertiesUtil.getProjectProperties().get("project.version");
 		String title = projectName + " " + projectVersion;
 
-		// If the version contains a "-" (e.g. "1.50.1-SNAPSHOT" or "1.50.1-beta1"), add a warning message
-		if (projectVersion.indexOf("-") > -1) {
-			title = title + " - " + Messages.getString("LooksFrame.26");
-		}
-
 		this.setTitle(title);
 		this.setDefaultCloseOperation(JFrame.HIDE_ON_CLOSE);
 		Dimension screenSize = getToolkit().getScreenSize();
