--- x264/encoder/set.c 2012-10-28 21:08:02.000000000 +0000
+++ x264/encoder/set.c 2012-10-28 21:10:55.146563398 +0000
@@ -575,8 +575,8 @@
 
     memcpy( payload, uuid, 16 );
     sprintf( payload+16, "x264 - core %d%s - H.264/MPEG-4 AVC codec - "
-             "Copy%s 2003-2012 - http://www.videolan.org/x264.html - options: %s",
-             X264_BUILD, X264_VERSION, HAVE_GPL?"left":"right", opts );
+             "Copy%s 2003-2012 - http://www.videolan.org/x264.html",
+             X264_BUILD, X264_VERSION, HAVE_GPL?"left":"right" );
     length = strlen(payload)+1;
 
     x264_sei_write( s, (uint8_t *)payload, length, SEI_USER_DATA_UNREGISTERED );
